multimedia/x265/Portfile

```
@@ -12,12 +12,12 @@ PortGroup           legacysupport 1.0
 legacysupport.newest_darwin_requires_legacy 9
 
 name                x265
-version             3.2.1
+version             3.4
```

Remove the `name` and `version` lines, using instead:
```suggestion
github.setup        videolan x265 3.4
```
And at the top of the Portfile, with the other PortGroup lines, add:
```
PortGroup           github 1.0
```